### DISCUSSION TITLE
Priorisiertes Batterieladen ohne Funktion!?

### DISCUSSION THREAD

**Discussion by Prideland** [camueller/SmartApplianceEnabler#596] [Allgemein]
Hi,
mir ist heute aufgefallen dass der SAE meinen Heizstab morgens gleich mit hoch fährt, obwohl der Akku noch nicht voll ist und in den Einstellungen vom SHM "Priorisiertes Batterieladen" aktiviert wurde.
Muss ich noch was anderes einstellen damit erst der Akku geladen wird und danach erst die Überschussleistung der PV-Anlage an meinen Heizstab geht?

![Akku_Vorrang](https://github.com/camueller/SmartApplianceEnabler/assets/138702513/0bbc9f2b-f859-4df8-90a0-c48aa2fd49ef)

![SHM_Prio_Batterie](https://github.com/camueller/SmartApplianceEnabler/assets/138702513/7fadafc4-4bef-492d-9756-d279092db79e)

**Comment by camueller** [comment]
Das regelt der Sunny Home Manager. Ob und wie Du beeinflussen kannst, dass das Laden des Akkus Priorirtät haben soll, kann ich nicht sagen. Am Besten ausprobieren oder in einem SMA-Forum/Support fragen.
